server.py

```diff
@@ -4,21 +4,26 @@
 import uvicorn
 from fastapi import FastAPI
 import sqlhelper
+import random
+import logging
+import time
+from args import get_args
 
 app = FastAPI()
+args = get_args()
 
 urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 http = urllib3.PoolManager(cert_reqs='CERT_NONE', assert_hostname=False)
 
-DB_FILE = 'parking.db'
+DB_FILE = None
```

Comment:
how about setting this to args.dbfile instead and using this variable in all the helper functions

Reply:
fixed